packages/app-elements/tailwind.config.cjs

```diff
@@ -24,8 +19,8 @@ module.exports = {
     },
     colors: {
       primary: {
-        light: transparentize(0.2, colorBrand),
-        DEFAULT: colorBrand,
+        light: 'rgba(102,110,255,0.8)', // transparentize(0.2, colorBrand),
+        DEFAULT: '#666EFF',
```

Comment:
I would keep the `colorBrand` variable for better readability.

We could use the `#RRGGBBAA` format for the `light` color.

```js
function alphaToHex(percentage) {
  return Math.floor((percentage * 255 / 100)).toString(16).toUpperCase().padStart(2, '0')
}
```

```js
{
  light: `${colorBrand}${alphaToHex(80)}`
}
```